extension/src/popup/components/mnemonicPhrase/ConfirmMnemonicPhrase.tsx

```diff
@@ -129,6 +121,10 @@ export const ConfirmMnemonicPhrase = ({
   const displaySelectedWords = () =>
     selectedWords.map((word) => convertToWord(word)).join(" ");
 
+  if (applicationState === APPLICATION_STATE.MNEMONIC_PHRASE_CONFIRMED) {
+    return <Redirect to={ROUTES.mnemonicPhraseConfirmed} />;
+  }
```

Comment:
This redirect was done with `history.push` in an action, but it was a little easier to test the outcome if I made it declarative like this.